src/ranged.cpp

```diff
@@ -968,13 +990,25 @@ static int print_ranged_chance( const player &p, const catacurses::window &w, in
     }
 
     if( display_type != "numbers" ) {
-        std::string symbols;
+        int column_number = 1;
+        if( !( panel_type == "compact" || panel_type == "labels-narrow" ) ) {
+            auto label = gettext( "Symbols:" );
+            mvwprintw( w, point( column_number, line_number ), label );
+            column_number += strlen( label ) + 1; // 1 for whitespace after 'Symbols:'
```

Comment:
`strlen` as well as `std::string::length` won't handle Unicode characters correctly. You need to use `utf8_width` instead.